x-pack/plugins/security_solution/public/assistant/use_anonymization_store/index.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useLocalStorage } from '../../common/components/local_storage';
+import { DEFAULT_ALLOW, DEFAULT_ALLOW_REPLACEMENT } from '../content/anonymization';
+import { LOCAL_STORAGE_KEY } from '../helpers';
+
+export interface UseAnonymizationStore {
+  defaultAllow: string[];
+  defaultAllowReplacement: string[];
+  setDefaultAllow: React.Dispatch<React.SetStateAction<string[]>>;
+  setDefaultAllowReplacement: React.Dispatch<React.SetStateAction<string[]>>;
+}
+
+const DEFAULT_ALLOW_KEY = `${LOCAL_STORAGE_KEY}.defaultAllow`;
+const DEFAULT_ALLOW_REPLACEMENT_KEY = `${LOCAL_STORAGE_KEY}.defaultAllowReplacement`;
+
+export const useAnonymizationStore = (): UseAnonymizationStore => {
```

Comment:
Once the anonymization state lives in the AssistantContextProvider we won't need this anymore